src/web/client/extension.ts

```diff
@@ -4,14 +4,22 @@
  * ------------------------------------------------------------------------------------------ */
 
 import * as vscode from "vscode";
+import TelemetryReporter from "@vscode/extension-telemetry";
+import { AI_KEY } from '../../client/constants';
+
+let _telemetry: TelemetryReporter;
 
 export function activate(context: vscode.ExtensionContext): void {
     console.log("Activated web extension!"); // sample code for testing the webExtension
+    // setup telemetry
+    _telemetry = new TelemetryReporter(context.extension.id, context.extension.packageJSON.version, AI_KEY);
+    context.subscriptions.push(_telemetry);
```

Comment:
you should add back the `Start` and `Stop` session events, like exists in the other VS Code `extension.ts`

Reply:
Also, the `activated` event is good to send too.